### DISCUSSION TITLE
Quick Question

### DISCUSSION THREAD

**Discussion by Arthura0121** [3kh0/ext-remover#426] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Quick question, if you downgrade ur chromebook, will it also unmanage it? cuz I wanna downgrade but keep it managed so I won't get in trouble.

**Comment by Aka-but-nice** [comment, marked as answer]
If you downgrade, depending on if you signed using your school email or your *home* email.

**Reply by Arthura0121** [reply]
> If you downgrade, depending on if you signed using your school email or your _home_ email.

so it stays managed??

**Reply by Aka-but-nice** [reply]
Well it depends, are you gonna log in with your home email or school one?

**Reply by Arthura0121** [reply]
> Well it depends, are you gonna log in with your home email or school one?

school account

**Reply by Arthura0121** [reply]
So I'm guessing my chromeOS will still be managed if I add my school account, correct? bc that's exactly what I wanted since I used to be able to use LTBEEF in ChromeOS 105

**Reply by Aka-but-nice** [reply]
Yes, it will be at 103 for example, but you'll still be on your school account.

**Comment by rwg100** [comment]
downgrading will get you in trouble anyways, since both are illegal

**Reply by Aka-but-nice** [reply]
> downgrading will get you in trouble anyways, since both are illegal

I searched the whole galaxy, and still couldn't find who tf asked

**Comment by piplup7575** [comment]
Downgrading should not affect management, but sh1mmer will.
